modules/ROOT/partials/glossary.adoc

```diff
@@ -0,0 +1,116 @@
+[glossary]
+
+[[action]]$action:: Freeform description of an action.
+[[alias]]$alias:: An alias name.
+[[alloc]]$alloc:: Allocator name.
+[[allocType]]$allocType:: Allocation type.
+[[auth]]$auth:: Auth provider name.
+[[boltMsgLenLimit]]$boltMsgLenLimit:: Bolt message length limit.
+[[boltServerState]]$boltServerState:: Bolt server state.
+[[cause]]$cause:: Freeform description of what caused the error.
+[[cfgSetting]]$cfgSetting:: Configuration setting key, for example, `server.https.enabled`.
+[[clause]]$clause:: A clause, for example, `USE`.
+[[cmd]]$cmd:: Cypher administrative command, for example, `DROP DATABASE`.
+[[component]]$component:: Component name.
+[[constr]]$constr:: Constraint name.
+[[constrDescrOrName]]$constrDescrOrName:: Constraint descriptor or name.
+[[context]]$context:: Freeform description of a context.
+[[coordinates]]$coordinates:: Coordinate value.
+[[count]]$count:: Amount
+[[countAllocs]]$countAllocs:: Desired number of servers to use.
+[[countSeeders]]$countSeeders:: Number of seeding servers.
+[[crs]]$crs:: Coordinate reference system, for example, WGS84.
+[[db]]$db:: Database name, for example, `myDb`.
+[[dim]]$dim:: Number representing index dimenionality.
+[[edition]]$edition:: Freeform edition description.
+[[endpointType]]$endpointType:: One of `start`, `end`.
+[[entityId]]$entityId:: ID of a node or a relationship.
+[[entityType]]$entityType:: A node or a relationship.
+[[expr]]$expr:: Cypher expression, for example, `1 + n.prop`.
+[[exprType]]$exprType:: Freeform expression type.
+[[feat]]$feat:: Freeform feature description.
+[[field]]$field:: Input field, for example, for a procedure or deserialization.
+[[format]]$format:: Duration format.
+[[fun]]$fun:: Function name.
+[[funClass]]$funClass:: User defined function implementation class name.
+[[graph]]$graph:: The name of a graph, for example, `myGraph`.
+[[graphTypeDependence]]$graphTypeDependence:: Graph type dependencency for constraint.
+[[hint]]$hint:: Freeform description of a "hint".
+[[hintList]]$hintList:: Comma-separated list of free form descriptions of hints.
+[[ident]]$ident:: Generic identifier.
+[[idx]]$idx:: Index name.
+[[idxDescr]]$idxDescr:: Index descriptor.
+[[idxDescrOrName]]$idxDescrOrName:: Index descriptor or name.
+[[idxOrConstr]]$idxOrConstr:: Index or constraint name
+[[idxOrConstrPat]]$idxOrConstrPat:: Index or constraint pattern
+[[idxType]]$idxType:: Index type, for example, text, vector, etc.
+[[input]]$input:: Piece of input.
+[[inputList]]$inputList:: Comma-separated list of inputs.
```

Comment:
When it comes to list parameters they are comma-separated, but they can also be configured to have any of ,/AND/OR between the second-to-last and last element.

Not sure if we want to mention this, given that can vary for the same parameter between different error codes

Reply:
I struggled a bit to imagine what the actual parameters would look like. I tried to improve the descriptions where possible, but I think we should go over all of them, be more specific, and give examples.

Reply:
Sounds good, just to give you an idea of what I mean for something I just implemented:

22N01, if the parameters are 
value = 42, 
valueTypeList = List.of(STRING, BOOLEAN, LIST<STRING>), 
valueType = INTEGER NOT NULL 

The message part of the status description will be
Expected the value 42 to be of type STRING, BOOLEAN **or**  LIST<STRING>, but was of type INTEGER NOT NULL.

This is because the valueList is configured to be a or-list here. For some other message it might be more suitable with e.g.

valid types are STRING, BOOLEAN **and**  LIST<STRING>